ui_framework/components/menu/menu.js

```diff
@@ -0,0 +1,39 @@
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+const getClassName = ({ className, contained = false }) =>
+  classNames('kuiMenu', className, {
+    'kuiMenu--contained': contained
+  });
+
+
+const KuiMenu = ({
+  contained,
+  className,
+  children,
+  ...rest
+}) => {
+  return (
+    <ul
+      className={getClassName({
+        className,
+        contained,
+      })}
+      {...rest}
+    >
+      {children}
+    </ul>
+  );
+};
+
+KuiMenu.propTypes = {
+  contained: PropTypes.bool,
+  className: React.PropTypes.string,
+  children:  PropTypes.node
+};
+
+export {
+  KuiMenu
```

Comment:
@cjcenizal Not sure about this: is this a pattern we're using other places? I think we should `export const KuiMenu` instead of listing exports like this.